pyproject.toml

```diff
@@ -3,6 +3,28 @@ profile = "black"
 multi_line_output = 3
 order_by_type = false
 
+[tool.mypy]
+disallow_incomplete_defs = true
+disallow_untyped_defs = true
+
+[[tool.mypy.overrides]] # Overrides for currently untyped modules
+module = [
+    "docs.*",
+    "docs.ext.*",
+    "gitlab.v4.objects.*",
+    "setup",
+    "tests.functional.*",
+    "tests.functional.api.*",
+    "tests.unit.*"
+]
+ignore_errors = true
+
+[[tool.mypy.overrides]] # Overrides to negate above patterns
+module = [
+    "gitlab.v4.objects.projects"
+]
```

Comment:
The overrides `module` listing does not match the previous `files` definition from `.mypy.ini`. Is this intentional?

Reply:
Yes, since `pre-commit` explicitly passes files to mypy (including sometimes `docs/`, `tests/`, I'm unsure why), I had to add some files outside `gitlab/` to ignores as well. Since it only supports wildcards, I then had to negate the `gitlab.v4.objects.*` ignore pattern with this one, since it's already typed.